rocky/tools/view_helpers.py

```diff
@@ -15,18 +15,19 @@ def convert_date_to_datetime(d: date) -> datetime:
     return datetime.combine(d, datetime.max.time(), tzinfo=timezone.utc)
 
 
-def get_mandatory_fields(request):
+def get_mandatory_fields(request, params: List[str] = []):
```

Comment:
Never use mutable default arguments. See [this](https://docs.python-guide.org/writing/gotchas/).

Reply:
I wanted to set a default and figured, since it's a list I could do `= []` but I'll use `= None` instead. Thanks for pointing this out.